chain/gssmrLatestTxnQueue/defaults.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package gssmr
+
+import (
+	"time"
+
+	"github.com/ChainSafe/gossamer/internal/log"
+	"github.com/ChainSafe/gossamer/lib/genesis"
+	"github.com/ChainSafe/gossamer/lib/runtime/wasmer"
+)
+
+var (
+	// GlobalConfig
+
+	// DefaultName Default node name
+	DefaultName = string("Gossamer")
+	// DefaultID Default chain ID
```

Comment:
Since this is just a temporary runtime to be deleted, and was just copied from the 9.20 branch, I think I will leave it here and then open an issue to update on other branches